deployability/modules/testing/tests/test_agent/test_stop.py

```diff
@@ -71,8 +71,7 @@ def test_service(wazuh_params):
         GeneralComponentActions.component_stop(agent_params, 'wazuh-agent')
 
     for agent_names, agent_params in wazuh_params['agents'].items():
-        assert 'inactive' in GeneralComponentActions.get_component_status(agent_params, 'wazuh-agent'), logger.error(f'{agent_names} is still active by command')
-        assert not GeneralComponentActions.isComponentActive(agent_params, 'wazuh-agent'), logger.error(f'{agent_names} is still active by command')
+        assert 'inactive' in GeneralComponentActions.get_component_status(agent_params, 'wazuh-agent') or 'not running' in GeneralComponentActions.get_component_status(agent_params, 'wazuh-agent'), logger.error(f'{agent_names} is still active by command')
```

Comment:
I don't recommend to call `logger.error` in assert statements. The second argument is the message assigned to the AssertionError exception. If you use `logger.error`, the return will be None, which will be the exception's message. Here's an example of what is the problem:

```
>>> import logging
>>> logger = logging.getLogger()
>>> assert False, logger.error('Error Message')
Error Message
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: None
>>> assert False, 'Error Message'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: Error Message
```

Is it required to log the error? Pytest shows all the failures in the output report.